tests/xmap_test.py

```diff
@@ -702,7 +696,7 @@ def testNestedMeshSPMD(self):
              axis_resources={'a': 'x', 'b': 'y'})
     xshape = (8, 2, 4, 5)
     x = jnp.arange(np.prod(xshape)).reshape(xshape)
-    y = f(x)
+    _ = f(x)
```

Comment:
Just do `f(x)`?

Reply:
Or maybe remove it? I wasn't clear if there was some required side-effect here for the test.

Reply:
If you want to remove it then maybe test it internally first but running the tests on tpu tfrt.